_api-reference/cat/cat-aliases.md

```diff
@@ -43,7 +43,7 @@ GET _cat/aliases
 
 All cat aliases URL parameters are optional.
```

Comment:
Not 100% sure about the usage here but shouldn't "cat" be capitalized since it's an acronym? "Compact and aligned text (CAT)"

Reply:
This was a discussion previously . . . the page title we use the Acronym, but the group decided to keep the page titles in left nav with "cat" lowercase because some areas have hardcoded "cat."

Reply:
Why don't we do the same for every API whether it's an acronym or not?